tests/multi_source/main.go

```diff
@@ -165,14 +167,35 @@ func truncateDDL(ctx context.Context, db *sql.DB) {
 func dml(ctx context.Context, db *sql.DB, table string, id int) {
 	var err error
 	var i int
-	var success int
-	sql := fmt.Sprintf("insert into test.`%s`(id) values(?)", table)
+	var insertSuccess int
+	var deleteSuccess int
+	insertSQL := fmt.Sprintf("insert into test.`%s`(id1, id2) values(?,?)", table)
+	deleteSQL := fmt.Sprintf("delete from test.`%s` where id1 = ? or id2 = ?", table)
 	for i = 0; ; i++ {
-		_, err = db.Exec(sql, i+id*100000000)
+		_, err = db.Exec(insertSQL, i+id*100000000, i+id*100000000+1)
 		if err == nil {
-			success++
-			if success%100 == 0 {
-				log.S().Info(id, " success: ", success)
+			insertSuccess++
+			if insertSuccess%100 == 0 {
+				log.S().Info(id, " insert success: ", insertSuccess)
+			}
+		}
+		if err != nil && !strings.HasPrefix(err.Error(), "Error 1146:") {
```

Comment:
In which case will `Error 1146:` happen?

Reply:
the `Error 1146:` will happen when the schema or table is dropped in case `createDropSchemaDDL`.